test/WelcomeStore.test.ts

```diff
@@ -168,4 +171,55 @@ describe("WelcomeStore.tagsErrorMessage", () => {
   });
 });
 
+describe("WelcomeStore.profileImageUrlErrorMessage", () => {
+  it("no error", () => {
+    //given
+    const mockFile = mock<File>();
+    when(mockFile.name).thenReturn("mock.png");
+    when(mockFile.size).thenReturn(MAX_IMAGE_BYTE_SIZE - 1);
+    const welcomeStore = new WelcomeStore();
+    expect(welcomeStore.profileImageUrlErrorMessage).toBeUndefined();
+
+    //when
+    welcomeStore.changeProfileImage(instance(mockFile));
+
+    //then
+    expect(welcomeStore.profileImageUrlErrorMessage).toBeUndefined();
+  });
+
+  it("should have error message profile image invalid extension", () => {
```

Comment:
문장에 when을 넣어주면 좋을 것 같네요
```suggestion
  it("should have error message profile when image invalid extension", () => {
```